### DISCUSSION TITLE
Problem with translations in Dashboard

### DISCUSSION THREAD

**Discussion by alonsoac** [LiveHelperChat/livehelperchat#2170] [General]
Hi. Updated from a version before Svelte. I see two problems in the dashboard. My personal language is English, the default site language is es_MX.  
1. With user language in English, some texts in the Dashboard are in Spanish, like the widget titles.
2. With user language in Spanish, some texts are in English, like 'All items will appear here.'  

I noted that there are some English texts embedded in design/defaulttheme/js/svelte/public/build/main.js . Does this mean all of those texts will not be translated?

**Comment by remdex** [comment]
Most likely they were just missed. Will be fixed with upcoming commits/releases.
